src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -416,144 +425,96 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
-        --current-temperature-position-top: 10%;
-        --current-temperature-text-padding-left: 15px;
-        --uom-font-size: 20px;
-        --uom-margin-left: -18px;
-        --current-mode-font-size: 18px;
-        --current-mod-margin-top: 6px;
-        --current-mod-margin-bottom: 12px;
-        --set-temperature-margin-bottom: -5px;
-      }
-      .medium {
-        --thermostat-padding-top: 20px;
-        --thermostat-margin-bottom: 20px;
-        --title-font-size: 23px;
-        --title-position-top: 27%;
-        --climate-info-position-top: 84%;
-        --set-temperature-font-size: 20px;
-        --current-temperature-font-size: 65px;
-        --current-temperature-position-top: 10%;
-        --current-temperature-text-padding-left: 15px;
-        --uom-font-size: 18px;
-        --uom-margin-left: -16px;
-        --current-mode-font-size: 16px;
-        --current-mod-margin-top: 4px;
-        --current-mod-margin-bottom: 4px;
-        --set-temperature-margin-bottom: -5px;
-      }
-      .small {
-        --thermostat-padding-top: 15px;
-        --thermostat-margin-bottom: 15px;
-        --title-font-size: 18px;
-        --title-position-top: 28%;
-        --climate-info-position-top: 78%;
-        --set-temperature-font-size: 16px;
-        --current-temperature-font-size: 55px;
-        --current-temperature-position-top: 5%;
-        --current-temperature-text-padding-left: 16px;
-        --uom-font-size: 16px;
-        --uom-margin-left: -14px;
-        --current-mode-font-size: 14px;
-        --current-mod-margin-top: 2px;
-        --current-mod-margin-bottom: 4px;
-        --set-temperature-margin-bottom: 0px;
-      }
-      .longName {
-        --title-font-size: 18px;
+
+      .more-info {
+        position: absolute;
+        cursor: pointer;
+        top: 0;
+        right: 0;
+        border-radius: 100%;
+        color: var(--secondary-text-color);
+        z-index: 25;
       }
-      #thermostat {
-        margin: 0 auto var(--thermostat-margin-bottom);
-        padding-top: var(--thermostat-padding-top);
-        padding-bottom: 32px;
+
+      #controls {
         display: flex;
         justify-content: center;
-        align-items: center;
+        padding: 16px;
+        position: relative;
       }
-      #thermostat round-slider {
-        margin: 0 auto;
-        display: inline-block;
+
+      #slider {
+        height: 100%;
+        width: 100%;
+        position: relative;
+        max-width: 300px;
+        min-width: 100px;
+      }
+
+      round-slider {
         --round-slider-path-color: var(--disabled-text-color);
         --round-slider-bar-color: var(--mode-color);
-        z-index: 20;
+        padding-bottom: 10%;
       }
-      #tooltip {
+
+      #slider-center {
         position: absolute;
-        top: 0;
-        left: 0;
-        right: 0;
-        height: 100%;
+        width: calc(100% - 40px);
+        height: calc(100% - 40px);
+        xborder: 1px green solid;
```

Comment:
Is `xborder` a typo or something I don't know?

Reply:
You're not keeping up with the latest properties?

Reply:
Let's call it a comment...